arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -560,9 +560,343 @@
+            "name": "NamespaceType",
+            "modelAsString": false
+          }
+        },
+        "createdAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets the exact time the message was created."
+        },
+        "DefaultMessageTimeToLive": {
+           "type": "string",
+           "format": "time-span",
+           "description": "Gets or sets the default message time to live value. This is the duration after which the message expires, starting from when the message is sent to Service Bus. This is the default value used when TimeToLive is not set on a message itself."
+          },
+         "DuplicateDetectionHistoryTimeWindow ": {
+           "type": "string",
+           "format": "time-span",
+           "description": "Gets or sets the TimeSpan structure that defines the duration of the duplicate detection history. The default value is 10 minutes.."
+          },
+          "EnableBatchedOperations": {
+          "type": "boolean",
+          "description": "Gets or sets a value that indicates whether server-side batched operations are enabled.."
+        },
+        "EnableDeadLetteringOnMessageExpiration": {
+          "type": "boolean",
+          "description": "Gets or sets a value that indicates whether this queue has dead letter support when a message expires."
+        },
+        "EnableExpress": {
+          "type": "boolean",
+          "description": "Gets or sets a value that indicates whether Express Entities are enabled. An express queue holds a message in memory temporarily before writing it to persistent storage."
+        },
+        "EnablePartitioning": {
+          "type": "boolean",
+          "description": "Gets or sets a value that indicates whether the queue to be partitioned across multiple message brokers is enabled."
+        },
+        "ForwardDeadLetteredMessagesTo": {
+          "type": "string",
+          "description": "Gets or sets the path to the recipient to which the dead lettered message is forwarded."
+        },
+        "ForwardTo": {
+          "type": "string",
+          "description": "Gets or sets the path to the recipient to which the message is forwarded."
+        },
+        "IsAnonymousAccessible": {
+          "type": "boolean",
+          "description": "Gets or sets a value that indicates whether the message is anonymous accessible."
+        },
+        "LockDuration ": {
+           "type": "string",
+           "format": "time-span",
+           "description": "Gets or sets the duration of a peek lock; that is, the amount of time that the message is locked for other receivers. The maximum value for LockDuration is 5 minutes; the default value is 1 minute."
+          },
+          "MaxDeliveryCount ": {
+           "format": "int32",
+           "type": "integer",
+           "description": "Gets or sets the maximum delivery count. A message is automatically deadlettered after this number of deliveries."
```

Comment:
You probably want to avoid using `Gets or sets` in descriptions; the language generators will automatically add that for things that get implemented as properties (in languages that support properties) and generate other appropriate comments for languages that use methods instead.